src/controller/org.controller/org.controller.js

```diff
@@ -69,6 +69,53 @@ async function getCna (req, res, next) {
   }
 }
 
+async function getUsers (req, res, next) {
+  try {
+    const shortName = req.ctx.org
+    const cnaShortName = req.ctx.params.shortname
+    const orgRepo = req.ctx.repositories.getOrgRepository()
+    const userRepo = req.ctx.repositories.getUserRepository()
+    const orgUUID = await orgRepo.getOrgUUID(cnaShortName)
+    let returned = false
+
+    if (!orgUUID) {
+      returned = true
+      logger.info(cnaShortName + ' CNA does not exist.')
+      return res.status(404).json(error.cnaDneParam(cnaShortName))
+    }
+
+    if (cnaShortName !== shortName && !returned) {
+      returned = true
+      logger.info({ message: cnaShortName + ' CNA can only be viewed by the owning CNA.' })
```

Comment:
We should be logging the req ID now that we have that on the context for log tracing.

Reply:
This should be addressed in a separate issue since this must be done in most of the logs throughout the services.